packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -25,6 +25,8 @@ import BlockListAppender from '../block-list-appender';
 import BlockInsertionPoint from './insertion-point';
 
 const innerToolbarHeight = 44;
+const headerToolBarHeight = 44;
+const blockBorderWidth = 1;
```

Comment:
I thought later and it seems gutenberg/packages/base-styles/_variables.scss file is a better place to declare these. $block-selected-border-width is already there. We can then use these constants to define some new styles in ./block-list/style.native.scss and import them here. 

Same for headerToolBarHeight, innerToolbarHeight. We can use new constants for these in both here and in their old code

Reply:
This makes a lot of sense. Good call!
Updated the code accordingly 👍